doc/bash-completion/lmms

```diff
@@ -89,7 +89,7 @@ _lmms()
     pars_render=(--float --bitrate --format --interpolation)
     pars_render+=(--loop --mode --output --profile)
     pars_render+=(--samplerate --oversampling)
-    actions=(dump render rendertracks upgrade)
+    actions=(dump compress render rendertracks upgrade)
     actions_old=(-d --dump -r --render --rendertracks -u --upgrade)
```

Comment:
The bash completion for compress works but it would also nice to have the old style --compress work.

Reply:
Right. None of the other actions are bash completing with old style longopts so scratch that for this PR.